### PR TITLE
30.get children info of parent

### PR DESCRIPTION
**Pull request by shagi9**

### CODE REVIEW

**Review by Karonuk — DISMISSED** [review]

**Review by Karonuk — DISMISSED** [review]

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -119,8 +121,11 @@ const routes: Routes = [
     canActivate: [AuthGuard, StudentGuard],
     children: pupilRoutes
   },
+
   { path: '404', component: PageNotFoundComponent },
   { path: '**', redirectTo: '404' },
+
+
```
Comment (DmMov) [inline src/app/app-routing.module.ts]: Dorou

File: src/app/components/children-info/children-info.component.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    Back
+</button>
+<yps-profile-card  
+    [user]="user"
+    *ngIf="user != null"
+>
+</yps-profile-card>
+
+
+
```
Comment (DmMov) [inline src/app/components/children-info/children-info.component.html]: Dorou

File: src/app/components/children-info/children-info.component.scss [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+        grid-template-columns: repeat(auto-fill, minmax(12rem, 1fr));
+        grid-gap: 1rem;
+        padding-right: 1rem;
+    }
+
+    .back-btn {
+        justify-self: start;
+    }
+}
+
```
Comment (DmMov) [inline src/app/components/children-info/children-info.component.scss]: Dorou

File: src/app/components/pupil-info-card/pupil-info-card.component.scss [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+      &:hover{
+        border: 1px solid $accent;
+        cursor: pointer;
+      }
+}
+
+  
+
+
```
Comment (DmMov) [inline src/app/components/pupil-info-card/pupil-info-card.component.scss]: Oooo dorou

File: src/app/app.module.ts [hunk 5]
```diff
@@ -58,6 +60,7 @@ import { ProfileCardComponent } from './components/profile-card/profile-card.com
 import { PupilProfileComponent } from './pages/pupil-profile/pupil-profile.component';
 import { PupilEventCardComponent } from './components/pupil-event-card/pupil-event-card.component';
 
+
```
Comment (DmMov) [inline src/app/app.module.ts]: Dorou

File: src/app/components/pupil-info-card/pupil-info-card.component.ts [hunk 6]
```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+import { ParentService } from 'src/app/services/parents/parent.service';
+import { ChildrenInfoComponent } from '../children-info/children-info.component';
+import { UserService } from 'src/app/services/user/user.service';
+import { Router } from '@angular/router';
+import { HttpClient } from '@angular/common/http';
+import { IUser } from 'src/app/models/IUser';
+
```
Comment (DmMov) [inline src/app/components/pupil-info-card/pupil-info-card.component.ts:4]: Dorou

**Review by vunt1k — APPROVED** [review]

**Review by DmMov — APPROVED** [review]

**Review by Karonuk — APPROVED** [review]

**Pull request merged**